### PR TITLE
AAE-9036 Refactoring to move identity management REST API to a new service

### PR DESCRIPTION
**Pull request by jesty**
This PR contains the required refactoring to move the new identity management REST API from the modeling service and the runtime bundle service to a new external service.

### CODE REVIEW

File: activiti-cloud-runtime-bundle-service/activiti-cloud-starter-runtime-bundle/src/main/java/org/activiti/cloud/starter/rb/configuration/ActivitiRuntimeBundleAutoConfiguration.java [hunk 1]
```diff
@@ -15,21 +15,17 @@
...
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Import;
-import org.springframework.context.annotation.PropertySource;
-
-import static org.activiti.cloud.starter.rb.configuration.ActivitiAuditProducerPartitionKeyExtractor.ACTIVITI_AUDIT_PRODUCER_PATITION_KEY_EXTRACTOR_NAME;
-import static org.activiti.cloud.starter.rb.configuration.ActivitiAuditProducerPartitionKeyExtractor.ACTIVITI_CLOUD_MESSAGING_PARTITIONED;
 
 @Configuration
 @Import(RuntimeBundleSwaggerConfig.class)
-@EnableIdentityManagementRestAPI
```
Comment (jesty) [inline activiti-cloud-runtime-bundle-service/activiti-cloud-starter-runtime-bundle/src/main/java/org/activiti/cloud/starter/rb/configuration/ActivitiRuntimeBundleAutoConfiguration.java:32]: Removed from Runtime Bundle and temporarily left on Modeling to don't break stuff in UI until they changed the code.

File: activiti-cloud-service-common/activiti-cloud-services-common-security/src/main/java/org/activiti/cloud/identity/model/UserApplicationAccess.java [hunk 2]
```diff
@@ -21,6 +21,10 @@ public class UserApplicationAccess {
     private String name;
     private List<String> roles;
 
+    public UserApplicationAccess() {
```
Comment (jesty) [inline activiti-cloud-service-common/activiti-cloud-services-common-security/src/main/java/org/activiti/cloud/identity/model/UserApplicationAccess.java:24]: required by integration tests

File: activiti-cloud-service-common/activiti-cloud-services-common-security/src/main/java/org/activiti/cloud/identity/model/UserRoles.java [hunk 3]
```diff
@@ -43,6 +43,9 @@ public static class UserGlobalAccess {
 
         private List<String> roles;
 
+        public UserGlobalAccess() {
```
Comment (jesty) [inline activiti-cloud-service-common/activiti-cloud-services-common-security/src/main/java/org/activiti/cloud/identity/model/UserRoles.java:46]: required by integration tests

**Comment by miguelruizdev** [comment]
after this refactoring, will there be an identity reference in the full URI?
I mean if  things like:   `<host>/<service>/v1/identity/groups?application=activiti` 
will turn into `<host>/identity/v1/groups?application=activiti`

**Comment by jesty** [comment]
> after this refactoring, will there be an identity reference in the full URI? I mean if things like: `<host>/<service>/v1/identity/groups?application=activiti` will turn into `<host>/identity/v1/groups?application=activiti`

No, because the new full URI is going to be like:

https://hxp-apa.envalfresco.com/identity-adapter-service/v1/groups?application=activiti

So the "identity" word is already in the URL.

**Comment by miguelruizdev** [comment]
Great, that's what I meant, if the `identity` reference will be now in the service, instead of on the path.
Understood 👌

**Review by miguelruizdev — APPROVED** [review]
LGTM

**Pull request merged**